### ISSUE TITLE
Python3 Support

### ISSUE THREAD

**Issue by WilliamMayor** [al45tair/vat#2]
Is there a stance on Python3 support?

I'm using Python3.4 and the library seems to work pretty well. So far the only complication I've seen is that the `RateCache` doesn't work because it does an `isinstance` comparing to `basestring`.

I'm happy to help if you can point me in a useful direction?

**Comment by al45tair** [comment]
Yes, it's supposed to work on Python 3, though that isn't how I have it deployed myself. If you spot problems with it, please feel free to submit pull requests or open issues.

**Comment by WilliamMayor** [comment]
I've managed to carve out some time today to work on this issue. I'll try to get a PR in this afternoon. Whilst I'm working on things, is there anything I can help out with?
